kiwi/system/prepare.py

```diff
@@ -496,7 +503,7 @@ def clean_package_manager_leftovers(self) -> None:
         )
         manager.clean_leftovers()
 
-    def _install_archives(self, archive_list):
+    def _install_archives(self, archive_list, archive_target_dir):
```

Comment:
The parameter should be imho named ```archive_target_dir_dict```, it's misleading otherwise. Alternatively I'm fine adding annotations even though it's a private method not considered to be used in the API